internal/pkg/node/transformers.go

```diff
@@ -276,8 +373,7 @@ func (node *NodeInfo) setterFrom(n *NodeConf, nameArg string,
 						} else if nestedInfoVal.Elem().Field(j).Type() == reflect.TypeOf(map[string](*Entry){}) {
 							confMap := nestedVal.Interface().(map[string]string)
 							if nestedInfoVal.Elem().Field(j).IsNil() {
-								ptr := nestedInfoVal.Elem().Field(j).Addr().Interface().(*map[string](*Entry))
```

Comment:
changes to setterFrom don't appear to relate to "added recursive getter function." They should likely be moved to the following commit.